posthog/models/cohort.py

```diff
@@ -115,6 +115,12 @@ def calculate_people(self, use_clickhouse=is_clickhouse_enabled()):
             self.save()
             capture_exception(err)
 
+    def calculate_people_ch(self):
```

Comment:
Q: Why is this not under `.calculate_people`? It introduces complexity below in various celery tasks, which feels weird.

Reply:
I thought of doing this but splitting it out would allow the tasks to be distributed across different workers I think so that we wouldn't pile up a few expensive queries on the same worker.

Reply:
Can you elaborate on this point - what's the plan with the other method, why do we still need it at all? Will we be getting rid of it at some point?

Reply:
Oh, I think the confusion is that I opted to manage the split of postgres vs clickhouse in `posthog/tasks/calculate_cohort.py` line 35. In that function, I'm calling the `calculate_cohort` function and `calculate_cohort_ch` so that we can split these two calculations into different tasks. 

Once we validate the cohorts are populated correctly and the surrounding querying is implemented I can make it only fire off the task respective to the deployment (clickhouse or postgres). Or alternatively but similarly, we can handle the clickhouse vs postgres difference in the function above instead of in the task. For now, I'm just keeping both because we're still testing the clickhouse table.